Based on the Board meeting (#115)
confidential-computing/governance: terminology/common-terminology.md

## Isolation Methodologies

The aforementioned packaging terms will be delivered by software running in concert with a TEE's hardware and firmware.  This combination will deliver a blend of data confidentiality, data integrity, and/or code integrity protections. 

> Data confidentiality is delivered in runtime by isolating a specific TEE context within the CPU (or other processing unit suchas a GPU), and potentially within the RAM.  There are multiple methodologies for delivering data-in-use protection. Three examples of technologies which limit the CPU’s addressability/reachability of TEE data include:

Reviewer: ```suggestion
Data confidentiality is delivered in runtime by isolating a specific TEE context within the CPU (or other processing unit such as a GPU), and potentially within the RAM.  There are multiple methodologies for delivering data-in-use protection. Three examples of technologies which limit the CPU’s addressability/reachability of TEE data include:
```

> Data confidentiality is delivered in runtime by isolating a specific TEE context within the CPU (or other processing unit such as a GPU), and potentially within the RAM.  There are multiple methodologies for delivering data-in-use protection. Three examples of technologies which limit the CPU’s addressability/reachability of TEE data include:

Reviewer 2: ```suggestion
Data confidentiality is delivered in runtime by isolating a specific TEE context within the processor (whether a CPU or other processing unit such as a GPU), and potentially within the RAM.  There are multiple methodologies for delivering data-in-use protection. Three examples of technologies which limit the CPU’s addressability/reachability of TEE data include:
```

* Paging control: non-TEE processes are not active within the CPU concurrently with TEE data.

Of course the CPU is not the only place where TEE data might be visible. Such data is commonly stored in RAM as well.  And when in RAM, side channel attacks might be attempted on the TEE data. Protecting the RAM from such attacks can be accomplished in ways such as the encryption of the TEE data within the RAM.  

> Chip vendors will typically bundle multiple isolation methodologies to protect their implementations. It is upon these bundles that software providing Confidential Computing capabilities is then layered.  

Reviewer: ```suggestion
Processor vendors will typically bundle multiple isolation methodologies to protect their implementations. It is upon these bundles that software providing Confidential Computing capabilities is then layered.  
```

Author: Looks good to me.

* Allow comparisons of packaged software alternatives, each of which may be based on different isolation methodologies.

> * Enable extensibility so that emerging chip technologies can assert packaging models they support (e.g., RISC-V based). 

Reviewer 2: ```suggestion
* Enable extensibility so that emerging processor technologies can assert packaging models they support (e.g., RISC-V based). 
```

Author: The proposed changes looks good to me
